index.html

```diff
@@ -6538,6 +6538,46 @@
       <code>sender.track.kind</code> is not "audio", throw an
       <code>InvalidParameters</code> exception.</p>
     </section>
+    <section id="rtcdtmf2sender-interface-definition*">
+      <h3>Interface Definition</h3>
+      <div>
+      <pre class="idl">[ Constructor (RTCRtpSender sender)]
+interface RTCDTMFSender {
```

Comment:
If I'm reading this correctly, it introduces `RTCDTMFSender` as a parallel type and not an alias of `RTCDtmfSender`?